docs/CONTRIBUTE.md

```diff
@@ -0,0 +1,55 @@
+This repository is the deployed version of the main repository of Software Discovery Tool as a submodule and the data repository as a sub submodule.
```

Comment:
I think what we want to explain here is bootstrapping. How does someone go from an empty VM, to using this deploy repository? It also may be worthwhile to link to a quick introduction to submodules, since I think most people will not be familiar with them.

So step one may be "Install according to the installation instructions for Ubuntu" but when you get to [Step 2](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Installation.md#step-2-checkout-the-source-code-into-opt-folder) use the following command instead:

sudo git clone --recurse-submodules https://github.com/openmainframeproject/software-discovery-tool-deploy.git
sudo ln -s /opt/software-discovery-tool-deploy/software-discovery-tool/ /opt/software-discovery-tool

And explain that this will check out all submodules, and then create the expected /opt/software-discovery-tool directory.